experimental/dds/tree2/src/feature-libraries/editable-tree-2/lazyEntity.ts

```diff
@@ -82,7 +82,13 @@ export abstract class LazyEntity<TSchema = unknown, TAnchor = unknown>
 		makePropertyNotEnumerable(this, anchorSymbol);
 	}
 
-	public abstract [Symbol.iterator](): Iterator<Tree>;
+	/**
+	 * Iterate through all nodes/fields in this node/field.
+	 *
+	 * @remarks
+	 * Concurrent modification is forbidden.
+	 */
+	public abstract boxedIterator(): IterableIterator<Tree>;
```

Comment:
its better to let this get its docs from the interface. Duplicating them makes maintaince harder, and makes it less clear this is implementing an interface.